jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "CircleCI API v2 introduction"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI Application Programming Interface (API) version 2, referred to as API v2 throughout this page, is a significant update to the existing CircleCI API v1.1 that is currently being used by internal and external audiences.
+
+* TOC 
+{:toc}
+
+## Introduction to API v2
```

Comment:
I think this section is nicely written. Nice job 👍

Reply:
Thanks :)